benchmark.py

```diff
@@ -0,0 +1,107 @@
+import gc
+import torch
+import time
+from tensorizer.serialization import TensorDeserializer
+from tensorizer.stream_io import CURLStreamFile
+import os
+import sys
+
+# Read in model name from command line, or env var, or default
+# to gpt-j-6b
+model_name = os.getenv("MODEL_NAME", "EleutherAI/gpt-j-6b/fp16")
```

Comment:
Seems like the model name is case sensitive since when I tested this I ran into a 403 error.
```suggestion
model_name = os.getenv("MODEL_NAME", "EleutherAI/gpt-j-6B/fp16")
```